mage_ai/data_preparation/templates/sensors/s3.py

```diff
@@ -0,0 +1,32 @@
+{% extends "testable.jinja" %}
```

Comment:
We might not need test for sensor block since return value is just true or false.